packages/auto-label/src/auto-label.ts

```diff
@@ -249,15 +245,102 @@ handler.autoLabelOnPR = async function autoLabelOnPR(
       logger.info(
         `Language label added to PR #${pull_number} in ${owner}/${repo} is ${language_label}`
       );
+      new_labels.push(language_label);
+    }
+  }
+
+  // Update all labels gathered so far if any
+  if (new_labels.length) {
+    logger.info(
+      `Labels added to PR# ${pull_number} in ${owner}/${repo} are ${JSON.stringify(
+        new_labels
+      )}`
+    );
+    await context.octokit.issues.addLabels({
+      owner,
+      repo,
+      issue_number: pull_number,
+      labels: new_labels,
+    });
+  }
+};
+
+/**
+ * Function updating a stale label on pull request based on configuration values.
+ * Removes previous staleness label if exists
+ */
+async function updateStalenessLabel(
+  context: Context<'pull_request'> | Context<'issues'>,
+  owner: string,
+  repo: string,
+  config: Config
+) {
+  // If user has turned on stale labels by configuring {staleness: {pullrequest: true, old: 60, critical: 120}}
+  // By default, this feature is turned off
+  if (!config.staleness?.pullrequest) {
+    return;
+  }
+  const response = await context.octokit.rest.pulls.list({
+    owner: owner,
+    repo: repo,
+  });
+  if (!response || !response.data) {
+    logger.info(`No active PRs available in ${owner}/${repo}...`);
+    return;
+  }
+
+  let old = config.staleness?.old || helper.DEFAULT_DAYS_TO_STALE;
+  let critical = config.staleness?.critical || helper.DEFAULT_DAYS_TO_STALE * 2;
+  let label = null;
+
+  // Make sure that config is right and if not, set defaults
+  if (old > critical || critical > helper.MAX_DAYS || old < 0) {
+    logger.info(
```

Comment:
I would be tempted to simply log an error here, and `return`, as it seems presumtuous to try to fix the value.

Then you can make `critical` and `old`, `const`, and it reads a bit better.

Reply:
Sure, will fix it